Add content about recurring corrections (#12915)
DataDog/documentation: content/en/monitors/service_level_objectives/_index.md

- For monitor-based SLOs, time in the correction window is not counted.
- For metric-based SLOs, all good and bad events in the correction window are not counted.

You have the option to create one-time corrections for ad-hoc adjustments, or recurring corrections for predictable adjustments that occur on a regular cadence. One-time corrections require a start and end time, while recurring corrections require a start time, duration, and interval. Recurring corrections are based on [iCalender RFC 5545's RRULE specification][14]. Specifying an end time for recurring corrections is optional in case you need the correction to repeat indefinitely. 

> For either type of correction, you must select a correction category that states why the correction is being made. The available categories are `Scheduled Maintenance`, `Outside Business Hours`, `Deployment`, `Other`. You can optionally include a description to provide additional context if necessary.

Reviewer: ```suggestion
For either type of correction, you must select a correction category that states why the correction is being made. The available categories are `Scheduled Maintenance`, `Outside Business Hours`, `Deployment`, and `Other`. You can optionally include a description to provide additional context if necessary.
```

Each SLO has a maximum limit of corrections that can be configured to ensure query performance. The limits per SLO are as follows:

| Correction Type   | Limit per SLO |
| ----------------- | ------------- |
| One-time          | 30            |
| Daily recurring   | 2             |
| Weekly recurring  | 3             |
| Monthly recurring | 5             |

> Status corrections are configurable through the UI, the [SLO status corrections API][12], or a [Terraform resource][13].

Reviewer: ```suggestion
You may configure status corrections through the UI, the [SLO status corrections API][12], or a [Terraform resource][13].
```

#### Access in the UI 

To access SLO status corrections in the UI:

1. Create a new SLO or click on an existing one.
2. Navigate to an SLO’s details side panel view.
3. Under the gear icon, select **Correct Status** to access the **Status Corrections** creation modal.

> 4. Choose between `One-Time` and `Recurring` in the **Select the Time Correction Window**, and specify the time period you which to correct.

Reviewer: ```suggestion
4. Choose between `One-Time` and `Recurring` in the **Select the Time Correction Window**, and specify the time period you wish to correct.
```
